cdap-client-tests/pom.xml

```diff
@@ -63,6 +63,12 @@ the License.
     </dependency>
     <dependency>
       <groupId>co.cask.cdap</groupId>
+      <artifactId>cdap-data-fabric</artifactId>
```

Comment:
I see. Would it be possible to move all tephra changes into a single commit, so that it is easy to revert later?